src/Socket_p.h

```diff
@@ -352,14 +364,14 @@ namespace Arcus
         uint32_t header = (ARCUS_SIGNATURE << 16) | (VERSION_MAJOR << 8) | (VERSION_MINOR);
         if(platform_socket.writeUInt32(header) == -1)
         {
-            error(ErrorCode::SendFailedError, "Could not send message header");
+            error(ErrorCode::SendFailedError, "Could not send message header 555");
             return;
         }
 
         uint32_t message_size = message->ByteSize();
         if(platform_socket.writeUInt32(message_size) == -1)
         {
-            error(ErrorCode::SendFailedError, "Could not send message size");
+            error(ErrorCode::SendFailedError, "Could not send message size 2222");
```

Comment:
Please remove the debug code here.
```suggestion
            error(ErrorCode::SendFailedError, "Could not send message size");
```